main/guides/getting-started/README.md

```diff
@@ -43,12 +48,29 @@ The Agoric SDK is supported on <a href="https://en.wikipedia.org/wiki/Linux">Lin
 - To launch a terminal on Linux, use the **Terminal** application.
 - To access WSL from Windows, visit the [WSL documentation](https://docs.microsoft.com/en-us/windows/wsl/).
 
-## Install Node.js 16.19.1 or Higher
+## Install Go
 
-Download Node.js from [Node.js](https://nodejs.org/) and follow the instructions for your platform. We recommend installing the LTS version of node 16.
+Download Go from [go.dev/doc/install](https://go.dev/doc/install) and follow the instructions for your platform.
 
 ```shell
-node --version # 16.19.1 or higher
+go version
+```
+
+## Install jq
+
+Download jq from [jqlang.github.io/jq/](https://jqlang.github.io/jq/) and follow the instructions for your platform.
+
+```shell
+jq --version
+```
+
+
+## Install Node.js 18.16.0 or Higher
```

Comment:
```suggestion
## Install Node.js LTS version such as 18.16.0
```

Reply:
This was changed to simply "Install Node.js". (The body of the section explains LTS versioning for Node.js)